lib/hydra/works/services/generic_file/full_text_extraction_service.rb

```diff
@@ -0,0 +1,33 @@
+module Hydra::Works
+  # Extract the full text from the content using Solr's extract handler
+  class FullTextExtractionService
+    def self.run(generic_file)
```

Comment:
Can you add lightweight YARD documentation to this public method since it's the main entry point to the service?